docker/Dockerfile

```diff
@@ -140,4 +139,10 @@ RUN if [ "$WITH_SUDO" = "yes" ]; then \
   echo "${nb_user} ALL=(ALL) NOPASSWD:ALL" >> /etc/sudoers \
 ;fi
 
+ARG BUILD_INFO=""
+RUN if [ -z "${BUILD_INFO}" ]; then \
+  BUILD_INFO="$(date)" \
+;fi \
```

Comment:
I think this will need an else, as the build arg won't be made into an environment variable will it?

Reply:
build arg is an env var, but if not supplied that populates it as default

Reply:
It's not automatically there at runtime. Or are you only intending to write it to the file?

https://vsupalov.com/docker-arg-env-variable-guide/

>ARG is only available during the build of a Docker image (RUN etc), not after the image is created and containers are started from it (ENTRYPOINT, CMD).